roles/container-engine/docker/molecule/default/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: delegated
+lint:
+  name: yamllint
+  # Disabled until issues are fixed
+  enabled: False
+platforms:
+  - name: kubespray-docker
+    image: quay.io/kubespray/vm-ubuntu-1804:latest
+    groups:
+      - k8s-cluster
+provisioner:
+  name: ansible
+  env:
+    ANSIBLE_ROLES_PATH: ../../../../
```

Comment:
Can we add ANSIBLE_CONFIG: ../../../../ansible.cfg so we include profile_tasks callback?

Reply:
Hm... I have to test this first. I have never used an external ansible.cfg in Molecule. I am not sure if Molecule works properly without its own configuration for Ansible.

Reply:
I configured Molecule to use profile_tasks in its own managed ansible.cfg. I hope that will suffice.

Reply:
perfect!